lstchain/calib/camera/calibrator.py

```diff
@@ -29,13 +29,11 @@ class LSTCameraCalibrator(CameraCalibrator):
 
     calibration_path = Unicode(
         '',
-        allow_none = True,
```

Comment:
why is this necessary here?

Reply:
the modification came from a comment from @MaxNoe ( https://github.com/cta-observatory/cta-lstchain/pull/331#discussion_r405317641)
Also I could found nowhere in the code where the None value was used.